styling-and-themes/custom-theme.md

```diff
@@ -11,12 +11,12 @@ position: 3
 
 # Custom Themes
 
-By customizing themes you can alter the default appearance of the Telerik components so they match the desired color scheme from your designers and fit with the rest of your site's coloring and style.
+By customizing themes, you can alter the default appearance of the Telerik components so they match the desired color scheme and the rest of your site's coloring and style.
 
 This article contains the following sections:
 
 * [What is the best way to customize the Telerik Blazor styles](#best-way-to-customize-the-telerik-blazor-css)
-* [Using the Sass Theme Builder](#using-the-sass-theme-builder)
+* [Using the Sass Theme Builder](#using-themebuilder)
 	* [Create New Theme](#create-new-theme)
 	* [Modifying Themes](#modifying-themes)
 	* [Import Custom Theme](#import-custom-theme)
```

Comment:
We don't need those anymore?

On a side note, can we link specific pages from the ThemeBuilder documentation site, which correspond to these three major use cases? This can make the transition easier.

Reply:
I will add the links.